apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss

```diff
@@ -0,0 +1,264 @@
+			-12px -338.66667px $color3,
+			220px -388.66667px $color3,
+			-69px -27.66667px $color5,
+			-111px -339.66667px $color1,
+			155px -237.66667px $color2,
+			-152px -380.66667px $color2,
+			-50px -37.66667px $color2,
+			-95px -175.66667px $color3,
+			-88px 10.33333px $color1,
+			112px -309.66667px $color1,
+			69px -415.66667px $color5,
+			168px -100.66667px $color5,
+			-244px 24.33333px $color6,
+			97px -325.66667px $color5,
+			-211px -182.66667px $color3,
+			236px -126.66667px $color4,
+			140px -196.66667px $color1,
+			125px -175.66667px $color2,
+			118px -381.66667px $color5,
+			144px -111.66667px $color6,
+			36px -78.66667px $color4,
+			-63px -196.66667px $color2,
+			-218px -227.66667px $color3,
+			-134px -377.66667px $color4,
+			-36px -412.66667px $color4,
+			209px -106.66667px $color3,
+			91px -278.66667px $color1,
+			-22px -191.66667px $color3,
+			139px -392.66667px $color3,
+			56px -2.66667px $color1,
+			-156px -276.66667px $color5,
+			-163px -233.66667px $color3,
+			-238px -346.66667px $color3,
+			62px -363.66667px $color1,
+			244px -170.66667px $color1,
+			224px -142.66667px $color2,
+			141px -208.66667px $color2,
+			211px -285.66667px $color5,
+			181px -128.66667px $color1,
+			90px -123.66667px $color2,
+			189px 70.33333px $color3,
+			-18px -383.66667px $color3,
+			100px -6.66667px $color5;
+	}
+
+	> .event-countdown__fireworks-after {
+		animation-delay: 1.25s, 1.25s, 1.25s;
+		animation-duration: 1.25s, 1.25s, 6.25s;
+	}
+}
+
+
+@-webkit-keyframes bang {
+	from {
+		box-shadow: 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white;
+	}
+}
+
+@keyframes bang {
+	from {
+		box-shadow: 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white, 0 0 white;
+	}
+}
+
+@-webkit-keyframes gravity {
+	to {
+		transform: translateY( 200px );
+		opacity: 0;
+	}
+}
+
+@keyframes gravity {
+	to {
+		transform: translateY( 200px );
+		-webkit-transform: translateY( 200px );
+		opacity: 0;
+	}
+}
+
+@-webkit-keyframes position {
```

Comment:
[Autoprefixer](https://github.com/postcss/autoprefixer) in CSS pipeline likely already adds vendor prefixes when needed, ok to remove them.

Reply:
:+1: Removed in https://github.com/Automattic/wp-calypso/pull/40926